samples/gazell/gzll_ack_payload/host/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: Gazell ACK payload host sample
+tests:
+  samples.gazell.gzll_ack_payload.host.build:
+    build_only: true
+    build_on_all: true
+    platform_allow: nrf52dk_nrf52832 nrf52833dk_nrf52833 nrf52840dk_nrf52840
```

Comment:
The same. Remove `build_on_all` and add `integration_platforms`

Reply:
Will do.